mayastor/src/bdev/nexus/nexus_io.rs

```diff
@@ -65,7 +65,7 @@ pub mod io_type {
     //    pub const IO_NUM_TYPES: u32 = 14;
 }
 
-/// the status of an IO
+/// the status of an IO - note: values copied from spdk bdev_module.h
 pub mod io_status {
     //pub const NOMEM: i32 = -4;
     //pub const SCSI_ERROR: i32 = -3;
```

Comment:
Should these commented out lines be removed?

Reply:
I think the idea is to un-comment them as the code that uses them gets implemented (they are from spdk). For example the reset IO recently. We may need to use the other error types at a later point when we need to support them.

Reply:
Ok :+1: